research/picai/fl_nnunet/config.yaml

```diff
@@ -1,7 +1,12 @@
-# You should set these yourself
+# FL4Health Settings:
 n_clients: 1
-nnunet_config: 2d
-# nnunet_plans: /home/shawn/Code/nnunet_storage/nnUNet_preprocessed/Dataset012_PICAI-debug/nnUNetPlans.json
 n_server_rounds: 1
-local_epochs: 2
-# starting_checkpoint: /home/shawn/Code/nnunet_storage/nnUNet_results/Dataset012_PICAI-debug/nnUNetTrainer_1epoch__nnUNetPlans__2d/fold_0/checkpoint_best.pth
+server_address: '0.0.0.0:8080'
+local_epochs: 3
+# local_steps: 50
+#starting_checkpoint: /home/shawn/Code/nnunet_storage/nnUNet_results/Dataset004_Hippocampus/nnUNetTrainer_1epoch__nnUNetPlans__2d/fold_0/checkpoint_best.pth
```

Comment:
Really dumb, but a space between the # and "starting_checkpoint" would be nice 🙂